spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/interpreter/InterpreterContext.scala

```diff
@@ -17,17 +17,47 @@ package za.co.absa.enceladus.conformance.interpreter
 
 import org.apache.spark.sql.SparkSession
 import org.apache.spark.sql.types.StructType
-import za.co.absa.enceladus.conformance.config.ConformanceConfig
+import org.apache.spark.storage.StorageLevel
+import za.co.absa.enceladus.conformance.config.{ConformanceConfig, ConformanceParser}
 import za.co.absa.enceladus.dao.MenasDAO
 import za.co.absa.enceladus.model.{Dataset => ConfDataset}
+import za.co.absa.enceladus.standardization_conformance.config.StdConformanceConfig
 
 /** Holds everything that is needed in between dynamic conformance interpreter stages */
-case class InterpreterContext (
-                                schema: StructType,
-                                conformance: ConfDataset,
-                                featureSwitches: FeatureSwitches,
-                                jobShortName: String,
-                                spark: SparkSession,
-                                dao: MenasDAO,
-                                progArgs: ConformanceConfig
-                              )
+
+case class InterpreterContextArgs(datasetName: String,
+                                  reportDate: String = "",
+                                  persistStorageLevel: Option[StorageLevel] = None)
+
+object InterpreterContextArgs {
+  def fromConformanceConfig[T](conformanceConfig: ConformanceParser[T]): InterpreterContextArgs = {
```

Comment:
'ConformanceParser` is really a misleading class name. It's not parsing Conformance, it's carrying (maybe parsing) Conformance configuration.

Reply:
Then ConformanceConfigParser?

Reply:
I like that. If considered too long, I don't have a problem with `ConfConfigParser`.
Btw, same with `StandardizationParser`